pkg/testers/ginkgo/ginkgo.go

```diff
@@ -172,6 +177,26 @@ func (t *Tester) validateLocalBinaries() error {
 	return nil
 }
 
+func (t *Tester) validateBinariesFromPath() error {
+	klog.V(2).Infof("checking for test binaries on PATH...")
+	for _, binary := range build.CommonTestBinaries {
+		path, err := stdexec.LookPath(binary)
+		if err != nil {
+			return fmt.Errorf("failed to validate binary %s from PATH: %w", binary, err)
+		}
+		klog.V(2).Infof("found existing %s at %s", binary, path)
+		switch binary {
+		case "e2e.test":
+			t.e2eTestPath = path
+		case "ginkgo":
+			t.ginkgoPath = path
+		case "kubectl":
+			t.kubectlPath = path
+		}
```

Comment:
This is kind of gross, I could use a `map[string]string` to store the binary paths instead of having a separate field in the `Tester` for them.